src/Context/ApiContext.php

```diff
@@ -331,6 +346,14 @@ public function getEnvironmentType()
     }
 
     /**
+     * @return string
+     */
+    public function getProxyUrlOrNull()
```

Comment:
We typically do not do OrNull in the getters in the SDK code. The reason is that the option of nullability is already stated in the doc header, and adding it into method name is not a normal practice among the PHP community.